src/language-html/options.js

```diff
@@ -32,4 +32,11 @@ module.exports = {
     default: false,
     description: "Indent script and style tags in Vue files.",
   },
+  bracketSameLine: {
```

Comment:
I think this should not in `html` but as a common option? I'm not sure how option category works, but seems weird that this option in HTML category, but effect `JSX`.

Reply:
I've moved it to the common options